src/neml2/models/solid_mechanics/Normality.cxx

```diff
@@ -79,9 +79,9 @@ Normality::set_value(bool out, bool dout_din, bool d2out_din2)
   {
     SolvingNonlinearSystem guard(false);
     if (out && !dout_din)
-      _model.dvalue();
```

Comment:
Probably want to leave a comment explaining these.

Reply:
This is actually a very good catch...